src/exportable/ExportableAlias.ts

```diff
@@ -12,4 +12,13 @@ export class ExportableAlias implements Exportable {
   toFSH(): string {
     return `Alias: ${this.alias} = ${this.url}`;
   }
+
+  static fromFSH(fshLine: string): ExportableAlias {
+    const regex = /^Alias: (?<alias>\S+) *= *(?<url>\S+).*/;
```

Comment:
This regular expression is a little different than the FSH grammar in terms of blank space: any whitespace characters may appear between the non-whitespace tokens, and at least one whitespace character must be used to separate the tokens. For example, the following is valid FSH for defining aliases:
```
Alias: $first =		http://example.org/first
Alias: $second =	http://example.org/second
Alias:	$third  =
					http://example.org/third
```
The set of characters ` \t\r\n\f\u00A0` are what FSH defines as whitespace characters. So, the regular expression should accept all of those. I think that `/Alias:\s+(?<alias>\S+)\s+=\s+(?<url>\S+).*/` should be close to what you want to use, but should be tested to make sure that it works correctly with all of the FSH whitespace characters.